go-fuzz/main.go

```diff
@@ -57,6 +58,16 @@ func main() {
 		log.Fatalf("both -http and -worker are specified")
 	}
 
+	if *flagDict != "" {
+		dictStat, err := os.Stat(*flagDict)
+		if err != nil {
+			log.Fatalf("cannot read dictionary file '%v'", *flagDict)
+		}
+		if dictStat.Size() == 0 {
```

Comment:
Any reason not to allow an empty dictionary? E.g. I can imagine someone using this to disable dictionaries. (Maybe it'd break something in the mutator code? If so, we could add an early exit when strLits is empty.)

Reply:
I feel if the user is manually specifying a dictionary, it would be strange if it was empty -- therefore probably a mistake. Happy to remove the `Fatalf` call, but I still think it should be logged. Thoughts?

Reply:
Sure, we could log.